src/v2/guide/installation.md

```diff
@@ -1,86 +1,83 @@
+  در هنگام توسعه اپلیکیشن ویو ما پیشنهاد می‌کنیم که [Vue Devtools](https://github.com/vuejs/vue-devtools#vue-devtools) را وری مرورگر خود نصب کنید که با محیط کاربری ساده‌اش به شما برای بررسی داده‌ها و خطایابی در اپلیکیشنتان کمک می‌کند.
 
-## Direct `<script>` Include
+## استفاده مستقیم از کد ویو با استفاده از تگ `<script>`
 
-Simply download and include with a script tag. `Vue` will be registered as a global variable.
+کافیست که کد ویو را دانلود کنید و آن را از طریق تگ script به صفحه خود وارد کنید. با این کار `Vue` به عنوان یک <span title='global variable'>متغییر سراسری</span> ثبت شده و در کد شما قابل استفاده خواهد بود. 
 
-<p class="tip">Don't use the minified version during development. You will miss out on all the nice warnings for common mistakes!</p>
+<p class="tip"> در هنگام توسعه از نسخه فشرده (minified) استفاده نکنید. چرا که خیلی از هشدارهای مفید در مورد خطاهای رایج را از دست خواهید داد!</p>
 
 <div id="downloads">
-  <a class="button" href="/js/vue.js" download>Development Version</a><span class="light info">With full warnings and debug mode</span>
+  <a class="button" href="/js/vue.js" download>Development Version</a><span class="light info">با تمام هشدارها و حالت خطایاب، مناسب در هنگام توسعه</span>
 
-  <a class="button" href="/js/vue.min.js" download>Production Version</a><span class="light info">Warnings stripped, {{gz_size}}KB min+gzip</span>
+  <a class="button" href="/js/vue.min.js" download>Production Version</a><span class="light info">هشدارهای کنسول خذف شذه، حجم فشرده {{gz_size}}KB min+gzip، مناسب برای محصول نهایی.</span>
 </div>
 
-### CDN
-
-For prototyping or learning purposes, you can use the latest version with:
+### استفاده از شبکه تحویل محتوا (CDN)
+برای نمونه اولیه و یا یادگیری ویو می توانید به جای دانلود کردن کد از لینک آخرین نسخه در شبکه تحویل محتوا استفاده کنید:
 
 ``` html
 <script src="https://cdn.jsdelivr.net/npm/vue"></script>
 ```
 
-For production, we recommend linking to a specific version number and build to avoid unexpected breakage from newer versions:
+برای محصول نهایی،‌ برای اجتناب از خطاهای و ناسازگاری‌های پیش بینی نشده،‌ ما پیشنهاد می کنیم که همیشه از لینک یک نسخه مشخص استفاده کنید:
 
 ``` html
 <script src="https://cdn.jsdelivr.net/npm/vue@2.6.0/dist/vue.js"></script>
 ```
-
-If you are using native ES Modules, there is also an ES Modules compatible build:
+اگر از ماژول‌های محلی ES استفاده می‌کنید، همچنین یک نسخه سازگار با ماژول‌های ES هم وجود دارد:
 
 ``` html
 <script type="module">
   import Vue from 'https://cdn.jsdelivr.net/npm/vue@2.6.0/dist/vue.esm.browser.js'
 </script>
 ```
 
-You can browse the source of the NPM package at [cdn.jsdelivr.net/npm/vue](https://cdn.jsdelivr.net/npm/vue/).
+شما می‌توانید منبع کد بسته NPM موجود در شبکه تحولی محتوا را در [cdn.jsdelivr.net/npm/vue](https://cdn.jsdelivr.net/npm/vue/) بررسی کنید.
```

Comment:
npm guys usually call it `npm` not `NPM`. But official docs is also using `NPM`, so ... :D

Reply:
So we can use npm here and also make a pull request to fix it in the official document as well ;)